src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -488,9 +488,13 @@ private function getType(string $type, bool $isCollection, string $className = n
      */
     private function computeDoc(Documentation $documentation, \ArrayObject $definitions, \ArrayObject $paths) : array
     {
+        /** @var Symfony\Component\Routing\RequestContext $requestContext */
+        $requestContext = $this->urlGenerator->getContext();
```

Comment:
This works, but is wrong as `UrlGenerator` doesn't have `getContext()`. What would be a better way to do it? /cc @dunglas